### ISSUE TITLE
VDirList vs IISApp

### ISSUE THREAD

**Issue by tfabraham** [BTDF/DeploymentFramework#405]
Hi, I just installed 5.6 and notice that the IISApp and IISAppPool settings are not interpreted like in version 6.0.

5.6 relies on the old VDirList. I really like the IISApp and IISAppPool setup but why is it gone?

Another eyebrow raiser is why the latest version is 5.6, where 6.0 is also available?

Am I missing something??

#### Attachments
[btdf.png](https://www.codeplex.com/Download/AttachmentDownload.ashx?ProjectName=biztalkdeployment&WorkItemId=11045&FileAttachmentId=5801)

#### This work item was migrated from CodePlex
CodePlex work item ID: '11045'
Vote count: '1'

**Comment by tfabraham** [comment]
[tfabraham@1/5/2016]
v6.0 (beta) was an attempt to take a big step forward, with significant changes to core functionality and major feature enhancements like fine-grained control over artifacts and a new IIS model.

A relatively small, but large enough to be very time-consuming, list of issues were filed against v6.0 beta.  As a result, I created v5.6 without the major, and potentially buggy, changes.  v5.6 picked up BizTalk 2013 R2 support and many of the smaller, but very useful, changes from v6.0.

Bottom line, I have to choose where I spend my very limited time.  v5.5 is an extremely stable release -- a great platform for v5.6, which I expect to be equally stable.  I'd rather have people use a solid, more easily supportable release (v5.6) than one with many unknowns (v6.0).  So v6.0 beta remains unsupported.

**Comment by tfabraham** [comment]
[jasperdefesche@1/6/2016]
ok, thanks for the clarification. I hope the IISApp and IISAppPool features will return someday though :-)

**Comment by tfabraham** [comment]
**Issue closed by _tfabraham_ with comment**
Responded

**Reason closed**
As Designed
